gomod.go

```diff
@@ -0,0 +1,7 @@
+package gomod_required_packages
```

Comment:
:warning: This file is required to include the `build-machinery-go` dependency, and it can be also used to add some "extra" dependencies that are not actually being used in our Go code, but we don't want it to be an indirect dependency or to be removed by the `go mod tidy`.

Reply:
OK, but I would not add it into file called `gomod.go`. I think introducing some new package as in console-operator (see https://github.com/openshift/console-operator/tree/master/pkg/dependencymagnet) is better.